php/libraries/File_Upload.class.inc

```diff
@@ -271,12 +271,7 @@ class File_Upload
             );
             return false;
         } else if (!file_exists($dest_dir) && $this->createDirectories) {
-            $config = NDB_Config::singleton();
             mkdir($dest_dir, 0775);
-            $grp = $config->getSetting('FileGroup');
```

Comment:
it seems like you'r deleting the config and the code that uses it. why is this being removed ?